config/test.json

```diff
@@ -1 +1,6 @@
-{}
+{
+  "db": {
+    "type": "better-sqlite3",
+    "database": "./testdb.sql"
```

Comment:
sqlite file extenetion should be `./testdb.sqlite`, `.sql` might indicate a SQL script file